Detections/SigninLogs/NRT_MFARejectedbyUser.yaml

```diff
@@ -0,0 +1,32 @@
+id: 3617d76d-b15e-4c6f-985e-a1dac73c592d
+name: NRT MFA Rejected by User
+description: |
+  'Identifies accurances where a user has rejected an MFA prompt. This could be an indicator that a threat actor has compromised the username and password of this user account and is using it to try and log into the account.
```

Comment:
probably a small typo in the word occurrences

Reply:
Fixed